crates/binjs_io/src/entropy/dictionary.rs

```diff
@@ -371,7 +389,8 @@ pub struct Dictionary<T> {
 impl<T> Dictionary<T> {
     /// Create a new dictionary using paths of `depth` depth
     /// and windows of `width` width.
-    pub fn new(depth: usize, width: usize) -> Self {
+    pub fn new(options: Options) -> Self {
+        let Options { depth, width } = options;
```

Comment:
If you don't need the original object, you can define destructuring directly in function definition.